godot-bevy/src/plugins/transforms/change_filter.rs

```diff
@@ -0,0 +1,8 @@
+use bevy::prelude::Component;
+
+/// Component that stores the tick when we last synced from Godot
+/// This allows us to detect changes that happened AFTER our sync
+#[derive(Component)]
+pub struct LastGodotSyncTick {
+    pub tick: u32,
```

Comment:
Would it make sense to use a type of `Tick` instead of `u32`. It would remove the need to call `.get()` for every time we compare with `change_tick.this_run()` and similar.